crates/notan_app/src/backend.rs

```diff
@@ -117,6 +117,12 @@ pub trait WindowBackend {
     /// Returns the window's size
     fn size(&self) -> (i32, i32);
 
+    /// Sets the window's position
+    fn set_position_pixels(&mut self, x: i32, y: i32);
```

Comment:
Do you think that it makes sense to remove `pixels` from the name and just name it `set_position` and `position` due that there is no other `set_position` method?

Reply:
The reason I went with 'pixels' is that it seems there's a `LogicalPosition` and `PhysicalPosition`. I think LP is by 'points' for scaled displays like on mac.  
But having said that I suppose most use cases would be pixels anyway and if LogicalPosition is implemented it can just be 'position_logical' or whatever.
Let me know if you'd still like me to change it.

Reply:
Come to think of it, size is by pixels, so it makes sense that position is assumed to be by pixels.